### DISCUSSION TITLE
request to join as student is giving me issues

### DISCUSSION THREAD

**Discussion by drewk312** [community/community#76529] [GitHub Education] [answered]
### Hi everyone,

Hello I'm trying to sign up for the student github I'm trying to  "If you don't have a student ID, or it doesn't include a date, you can upload a letter on school letterhead or any documentation with a date that demonstrates your current enrollment. " But where there should be somewhere to upload a document or attachment i have {:class=>"discount_request_id", :data=>{:discount_request_id=>12689055}} and i tried to upload a photo of my virtual ID but it keeps getting rejected where i can send my info?

**Comment by thaonguyen2417** [comment, marked as answer]
Hey there,
What I did is taking a screenshot of the offer letter and upload it to where you're supposed to upload the ID.
Try and hope it works for you :)

**Reply by drewk312** [reply]
thank you so much!

**Comment by aliyaalias19** [comment]
Hi, I'm experiencing the same issue. I'm unsure about what I should do. Any suggestions would be greatly appreciated.
![image](https://github.com/community/community/assets/133945198/45b076fd-0982-454b-86b9-f776c29157ac)

I attempted to use a different browser, but it did not display the upload button for me.
![image](https://github.com/community/community/assets/133945198/64414016-9933-4890-bec8-473e0133fda0)

**Reply by drewk312** [reply]
so what i ended up doing is dragging my jpeg file into the take photo and it processed it and took that, so maybe try that!?

**Reply by aliyaalias19** [reply]
It works. Thank a lot!!
